build/release_specs/cli-rbc-spec.json

```
@@ -0,0 +1,12 @@
+{
+  "files": [
+    {
+      "pattern": "ecosys-jfrog-cli/v2/scripts/getCli.sh",
+      "target": "jfrog-cli/v2/scripts/getCli.sh"
+    },
+    {
+      "pattern": "ecosys-jfrog-cli/v2/${VERSION}/(*)/(*)",
+      "target": "jfrog-cli/v2/${VERSION}/{1}/{2}"
```

Before the release (Due to https://github.com/jfrog/jfrog-cli-core/pull/246):
```suggestion
      "pattern": "ecosys-jfrog-cli/(v2/${VERSION}/*/*)",
      "target": "jfrog-cli/{1}"
```

After the release can be even shorter:
```suggestion
      "pattern": "ecosys-jfrog-cli/(v2/${VERSION}/*)",
      "target": "jfrog-cli/{1}"
```